lib/matplotlib/axes/_secondary_axes.py

```diff
@@ -254,9 +235,7 @@ def draw(self, renderer=None, inframe=False):
         using the converter specified by
         `~.axes._secondary_axes.set_functions` (or *functions*
         parameter when axes initialized.)
-
         """
```

Comment:
By curiosity, is there a reason for removing the blank line after the docstring? It looks like the file already hosts a mix of “blank line”/“no blank line” after a docstring anyway…

Reply:
See https://github.com/PyCQA/pydocstyle/issues/361 for how I tend to use blank lines (basically, use one if the body has other blank lines).  Not religious about that, but given that you asked, I made the changes throughout the module :)